src/app/components/list/list.component.scss

```diff
@@ -1,73 +1,156 @@
 @import '../../../styles/variables.scss';
 
 // la content, nu ii pot da un height % ca sa-l fac responsive. am fortat cu min si max height in px
+// .c-list {
+//   display: flex;
```

Comment:
Do we need this commented code?

Reply:
Thanks. I removed it now. It was css from before changing the list component to be different based on users that I forgot to remove.